README.md

```diff
@@ -1,24 +1,40 @@
+This plugin allows the forwarding of uploaded files to an antivirus scanning application and prevents the upload from completing if there is a virus detected in the file. Use it to prevent users from inadvertently spreading malware or viruses via your Mattermost instance. 
 
-Currently the plugin supports [ClamAV anti-virus software](https://www.clamav.net/) across browser, Desktop Apps and the Mobile Apps.
+Currently the plugin supports [ClamAV anti-virus software](https://www.clamav.net/) across browser, Desktop Apps and the Mobile Apps. ClamAV is an open source (GPL) anti-virus engine used in a variety of situations including email scanning, web scanning, and end point security. It provides a number of utilities including a flexible and scalable multi-threaded daemon and an advanced tool for automatic database updates. A ClamAV server can be easily provisioned as a docker container that runs alongside Mattermost. 
 
-**Supported Mattermost Server Versions: 5.2+**
+**Requirements:**
+
+- Mattermost Server Version: 5.2+
+- ClamAV Server access
 
 ## Installation
 
 1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-antivirus/releases) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost System Console under **System Console > Plugins > Management** to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Install ClamAV (clamd) for virus scanning. One option is to install with docker:
+2. In the Mattermost System Console under **System Console > Plugins > Management** upload the file to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+3. Install ClamAV (clamd) for virus scanning. One easy option is to provision a ClamAV container with docker by running the following command.  Assuming you have already installed Docker, this will download and install the latest version of ClamAV and setup a server with an open port at 3310:
 
    ```
    docker run -d -p 3310:3310 mkodockx/docker-clamav
    ```
 
-4. Once clamd is running, configure the plugin to make requests to your clamd instance. Go to **System Console > Plugins > Antivirus** and configure **Clamav Host and Port** to point at your clamd instance.
-5. Activate the plugin at **System Console > Plugins > Management**.
+4. Once clamd server is running, configure the plugin to make requests to your clamd instance. Go to **System Console > Plugins > Antivirus** and configure **Clamav Host and Port** to point at your clamd instance.  
+5. Activate the plugin at **System Console > Plugins > Management** and ensure that it starts with no errors
```

Comment:
```suggestion
5. Activate the plugin at **System Console > Plugins > Management** and ensure it starts with no errors.
```

Reply:
If they do hit errors, should we recommend they check **System Console > Reporting > Server Logs** for details?